src/variables.scss

```diff
@@ -20,6 +20,7 @@ $l-font-size-mobile: 42px;
 $xl-font-size-mobile: 92px;
 
 // breakpoints
+$s-breakpoint: 360px;
```

Comment:
why we need variable with this resolution?
In my opinion we should start styling views from mobile to desktop (mobile first). I think that users on watches etc will not play this game.